README.md

```diff
@@ -1 +1,66 @@
-# LeaderBoard-
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# LeaderBoard
+
+> API TBD
+
+## Built With
+
+- HTML, CSS and Javascript
+- Git and Github
+
+
+## Live Demo (if available)
+
+https://emmykage.github.io/
+
+
+## Getting Started
+
+
+
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+  None
+
```

Comment:
**[REQUIRED CHANGES]**

- Please, kindly update or remove this header as you've specified there is no prerequisite needed to run the project.
- Also, I think the user could need a visual studio code or any other code editor they may prefer.